test/e2e/awsiam_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
```

Comment:
can we remove these changes?
Probably added by a formater?

Reply:
Yeah i ran gofumpt and it made these changes.